client/v2/go.mod

```diff
@@ -5,7 +5,7 @@ go 1.15
 require (
 	github.com/json-iterator/go v1.1.10
 	github.com/modern-go/reflect2 v1.0.1
-	go.etcd.io/etcd/api/v3 v3.0.0-20201012212543-0b95e8cef14a
+	go.etcd.io/etcd/api/v3 v3.0.0-00010101000000-000000000000
```

Comment:
Not sure why the v2 go.mod and go.sum are changed?

Reply:
It was overlooked in previous CLs that not all the versions stay at consistent version. 
When all modules are split, IMHO we should tag them all (so change also here) to v3.5.0-pre

Before: 
```
% git grep go.etcd.io/etcd | grep "v3.0.0" 
client/v2/go.mod:	go.etcd.io/etcd/api/v3 v3.0.0-20201012212543-0b95e8cef14a
client/v2/go.mod:	go.etcd.io/etcd/pkg/v3 v3.0.0-00010101000000-000000000000
go.mod:	go.etcd.io/etcd/api/v3 v3.0.0-20201012212543-0b95e8cef14a
go.mod:	go.etcd.io/etcd/pkg/v3 v3.0.0-00010101000000-000000000000
go.mod:	go.etcd.io/etcd/raft/v3 v3.0.0-00010101000000-000000000000
raft/go.mod:	go.etcd.io/etcd/pkg/v3 v3.0.0-00010101000000-000000000000
tests/go.mod:	go.etcd.io/etcd/api/v3 v3.0.0-20201012212543-0b95e8cef14a
tests/go.mod:	go.etcd.io/etcd/pkg/v3 v3.0.0-00010101000000-000000000000
tests/go.mod:	go.etcd.io/etcd/raft/v3 v3.0.0-00010101000000-000000000000
tests/go.mod:	go.etcd.io/etcd/v3 v3.0.0-00010101000000-000000000000
```
After:
```
% git grep go.etcd.io/etcd | grep "v3.0.0"       
client/v2/go.mod:	go.etcd.io/etcd/api/v3 v3.0.0-00010101000000-000000000000
client/v2/go.mod:	go.etcd.io/etcd/pkg/v3 v3.0.0-00010101000000-000000000000
client/v3/go.mod:	go.etcd.io/etcd/api/v3 v3.0.0-00010101000000-000000000000
client/v3/go.mod:	go.etcd.io/etcd/pkg/v3 v3.0.0-00010101000000-000000000000
go.mod:	go.etcd.io/etcd/api/v3 v3.0.0-00010101000000-000000000000
go.mod:	go.etcd.io/etcd/client/v3 v3.0.0-00010101000000-000000000000
go.mod:	go.etcd.io/etcd/pkg/v3 v3.0.0-00010101000000-000000000000
go.mod:	go.etcd.io/etcd/raft/v3 v3.0.0-00010101000000-000000000000
raft/go.mod:	go.etcd.io/etcd/pkg/v3 v3.0.0-00010101000000-000000000000
tests/go.mod:	go.etcd.io/etcd/api/v3 v3.0.0-00010101000000-000000000000
tests/go.mod:	go.etcd.io/etcd/client/v3 v3.0.0-00010101000000-000000000000
tests/go.mod:	go.etcd.io/etcd/pkg/v3 v3.0.0-00010101000000-000000000000
tests/go.mod:	go.etcd.io/etcd/raft/v3 v3.0.0-00010101000000-000000000000
tests/go.mod:	go.etcd.io/etcd/v3 v3.0.0-00010101000000-000000000000
```